### DISCUSSION TITLE
Machine Runs Away After Jogging

### DISCUSSION THREAD

**Discussion by andrewccard** [winder/Universal-G-Code-Sender#2125] [Help] [answered]
I just got my limit switches setup and homing works exactly as intended. However now there is an issue with jogging. Whenever i send a jog command the machine runs in the Y- Direction and will not stop. Pressing the stop button doesn't work because it is spamming the console with gcode. The only way to stop it is by unplugging the comm or power cable. Ive never had this happen before.

**Comment by breiler** [comment, marked as answer]
What OS are you running on?
Which version of UGS and what type and version of controller are you using?

Have you mapped any keyboard shortcut or connected a gamepad?

Logs may also be helpful:
https://github.com/winder/Universal-G-Code-Sender/wiki/Configuration#log-files

**Reply by andrewccard** [reply]
Windows 10
I redownloaded today so most likely latest version of UGS
The controller is a cnc shield on my mill right carve king 1

As for key binds it should all be completely default. All i have gone through was the machine wizard to get homing to work.


[messages.txt](https://github.com/winder/Universal-G-Code-Sender/files/10422507/messages.txt)

**Reply by breiler** [reply]
What version does your machine report if you type $I in the console?

**Reply by andrewccard** [reply]
[VER:1.1f.20170131:]
[OPT:SL, 15, 128]
ok

**Reply by breiler** [reply]
Can you post on of the gcode command that is sent for jogging?

**Reply by andrewccard** [reply]
I can get a specific one tomorrow but im almost positive it was something like

Jog: (Y-) -1.667

but in gcode of course. the console was flooded with that and ok.

**Reply by andrewccard** [reply]
hitting stop didnt work because it was being spammed it wasnt just one big jog command

**Reply by andrewccard** [reply]
Also to clarify the machine will run the sent jog command before running to Y-, and you can even send jogs between the spam and they run fine

**Reply by andrewccard** [reply]
Apparently the ghost in my machine is gone because after going to get the exact gcode for you the issue is no longer present. If it shows up again ill come back but for now the issue is solved. Do you have any idea what might have caused this in the first place?

**Reply by breiler** [reply]
No, no idea. =/

**Comment by pixelwaster** [comment]
Not to hijack the thread, I had the same thing happen to me. UGS 2.0.15, WIN 10, Woodpecker Board running grblr 1.1f. The command before trying run off the X axis was "cls". I know that cls isn't a real command, but I thought I was using a different sender. I tried to jog X -10mm, but as soon as it was done (10mm), it was back to running the other way. Stop didn't work, but pause did pause the runaway X-axis until any other command (even a carriage return) was entered or button pushed. 

The line in the log after 'cls' and every good command was '... responding to an unknown command'. I would post the log but it isn't there anymore. I also cannot reproduce the effect.

**Reply by breiler** [reply]
Keep a lookout and post the console log if this happens again. I am "hoping" that this is caused by interferrence on the usb serial line and not a program error in UGS. If this does happen again, check your USB cable and make sure it is shielded and of good quality.
